src/olympia/blocklist/admin.py

```diff
@@ -94,6 +99,13 @@ def _multi_input_view(self, request, *, add, form_url='',
         return TemplateResponse(
             request, 'blocklist/multi_guid_input.html', context)
 
+    def add_from_addon_pk_view(self, request, pk, **kwargs):
+        addon = get_object_or_404(Addon.unfiltered, pk=pk or kwargs.get('pk'))
+        get_params = [f'{key}={value}' for key, value in request.GET.items()]
+        return redirect(
+            reverse('admin:blocklist_blocklistsubmission_add') + '?' +
+            '&'.join([f'guids={addon.guid}'] + get_params))
```

Comment:
Would break if somehow passed `GET` params that need to be escaped. How about:

```suggestion
        get_params = request.GET.urlencode()
        return redirect(
            reverse('admin:blocklist_blocklistsubmission_add') +
            f'?guids={addon.guid}&{get_params}'
```

Reply:
https://github.com/mozilla/addons-server/pull/13898/commits/d8357eca085cb7ad177281afbb897ce432360802 (the suggestion was missing the closing `)`